sdk/src/error.rs

```diff
@@ -11,3 +11,11 @@ pub enum ActorDeleteError {
     #[error("deletion beneficiary does not exist")]
     BeneficiaryDoesNotExist,
 }
+
+#[derive(Copy, Clone, Debug, Error)]
+pub enum EpochBoundsError {
```

Comment:
(yeah, yeah, here come the rust police)

Basically, this way we can distinguish between "you gave us a bad epoch" and "I can't do that dave".

Reply:
I dont think anyone is going to distinguish that programmatically, the error message is what matters most.

In theory it _could_ be useful, in practice we'll just `?` raise it.

Reply:
Well, I have three alternatives:

- Panic in the sdk method.
- Return an Option with no additional information.
- Return a Result with a syscall error number.

Reply:
yeah, lets just return some detail... maybe it will be useful one day :)